src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/nochangeincurrentposition/NoPositionChangeElasticSearchQuery.java

```diff
@@ -1,19 +1,22 @@
 package uk.gov.hmcts.ethos.replacement.docmosis.reports.nochangeincurrentposition;
 
-import org.elasticsearch.index.query.RangeQueryBuilder;
 import org.elasticsearch.search.builder.SearchSourceBuilder;
 
-import static org.elasticsearch.index.query.QueryBuilders.boolQuery;
+import java.time.LocalDate;
+
+import static org.elasticsearch.index.query.QueryBuilders.rangeQuery;
 import static uk.gov.hmcts.ecm.common.model.helper.Constants.MAX_ES_SIZE;
+import static uk.gov.hmcts.ecm.common.model.helper.Constants.OLD_DATE_TIME_PATTERN2;
 
 class NoPositionChangeElasticSearchQuery {
     private NoPositionChangeElasticSearchQuery() {
         // Access through static methods
     }
 
     static String create(String reportDateLimit) {
-        var boolQueryBuilder = boolQuery()
-                .filter(new RangeQueryBuilder("data.dateToPosition").lte(reportDateLimit).includeUpper(false));
+        // using EPOCH/UNIX date as lower limit for search query.
+        var lowerDateLimit = LocalDate.of(1970, 1, 1).format(OLD_DATE_TIME_PATTERN2);
```

Comment:
Why are you searching since 1970? If you want data for last 3 months, shouldn't the lower date be the date 3 months ago?

Reply:
Report is the opposite, we don't want the data for the last 3 months, we want the data from the start upto 3 months ago. If they user had the date 31/03/2022, the report should show cases from the beginning (01/01/1970 using Angela's logic) up until 31/12/2021.

Reply:
ahh yes!